Server/src/Server.Infrastructure/Persistence/BookBookDbContext.cs

```diff
@@ -9,14 +10,14 @@ internal sealed class BookBookDbContext : DbContext
     public DbSet<Book> Books { get; set; }
     public DbSet<BookCategory> BookCategories { get; set; }
     public DbSet<Publisher> Publishers { get; set; }
+    public DbSet<Identity> Identities { get; set; }
+    public DbSet<Session> Sessions { get; set; }
     public DbSet<Library> Libraries { get; set; }
-    public DbSet<Address> Addresses { get; set; }
-    public DbSet<OpenHours> OpenHours { get; set; }
```

Comment:
Hmm, czemu to usunięte?

Reply:
o dobrze że zauważyłeś za dużo się usunęło podczas fixania konfliktów XD